x-pack/plugins/security_solution/public/entity_analytics/components/asset_criticality_file_uploader/helpers.ts

```diff
@@ -49,3 +50,12 @@ export const buildAnnotationsFromError = (
 
   return annotations;
 };
+
+export const formatScheduledRunTime = (time: string | undefined): string => {
```

Comment:
An open question, if a util is only used by one component would it be clearer to have it in the component file? Or do you prefer to have just the component in a tsx file

Reply:
If the function logic is self-contained, I prefer to remove the function from the component and make it public so we can easily write unit tests. That makes the component smaller and easier to understand. But this function name `formatScheduledRunTime`  is very specific and could be more generic like `formatTimeFromNow`.
I hope we can reuse it on the Risk Engine management page.